platform/src/main/java/org/stellar/anchor/platform/service/HealthCheckContext.java

```diff
@@ -0,0 +1,3 @@
+package org.stellar.anchor.platform.service;
+
+public interface HealthCheckContext {}
```

Comment:
What do we need this for? Seems like it could be deleted.

Reply:
The `HealthCheckContext` was for providing the context for each health check, such as environment variables, configuration profiles, etc. However, it is not used at this moment. 

I had removed the interface for now. We can add it back when needed.